x-pack/plugins/fleet/common/services/index.ts

```diff
@@ -80,3 +80,10 @@ export {
   isAgentPolicyValidForLicense,
   unsetAgentPolicyAccordingToLicenseLevel,
 } from './agent_policy_config';
+
+export { createSoFindIterable } from './create_so_find_iterable';
```

Comment:
Should this be promoted into a package instead of exporting it from Fleet plugin?
What do you think @paul-tavares

Reply:
We could. I actually just copied the ES utility to Security solution in the PR I am working on. Security Solution has many packages - I guess we could add it to one of them (I think there even a elastic search helper package for security solution). 

I don't think we should be exporting generic utilities like this one from Fleet and importing it from there. The exports of a Plugin should be to support their published interfaces not to re-use their internal code. 

my vote: just copy this utility to our own `endpoint/utils/`